src/dappsExplorer/FavoriteDappsSectionSearch.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { favoriteDappsSelector } from 'src/dapps/selectors'
+import { ActiveDapp, DappSection } from 'src/dapps/types'
+import DappCard from 'src/dappsExplorer/DappCard'
+import NoResultsSearch from 'src/dappsExplorer/NoResultsSearch'
+import { calculateSearchScore } from 'src/dappsExplorer/utils'
+import StarIllustration from 'src/icons/StarIllustration'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  onPressDapp: (dapp: ActiveDapp) => void
+  searchQuery: string
+  setFavoriteResultsEmpty: (empty: boolean) => void
+}
+
+export function FavoriteDappsSectionSearch({
```

Comment:
i'm starting to wonder about the value of this favourite dapps abstraction :/ i think it makes sense with the existing dapps screen because the "all" dapps section is an actual section list with categories as sections, whereas the favourites is a list of cards.

but with the filters and search, it's no longer the same section list. the favourites and all sections are now pretty much the same component, with different labels and placeholders and cards....i wonder if we should combine them. this is not a very well thought out suggestion yet, i just thought of it because i notice that we're doing the search logic both here and in the main dapps screen :/ and now on closer inspection, much of the rendering code is similar too.